pkg/systemd/ssh-list-host-keys.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+set -eu
+port="22"
+host="localhost"
+
+parse_addr() {
```

Comment:
So I guess the goal of all this complexity is to make this code work as non-root right?